python/python/ecl/util/__init__.py

```diff
@@ -91,6 +91,34 @@ def __init__(self, prototype, bind=True):
 ###  See https://github.com/Statoil/libecl/issues/142 for a discussion and for
 ###  usage.
 ###
+
+import os
+__cc = os.environ.get('ECLWARNING', None)  # __cc in (None, 'user', 'dev', 'hard')
```

Comment:
OK - this looks good; just one question: is there any semantic difference between `user` and `dev`?